tests/structure/test_prototype.py

```diff
@@ -0,0 +1,100 @@
+from typing import Final
+
+import pytest
+from pymatgen.core.structure import Lattice, Structure
+
+from matbench_discovery import TEST_FILES
+from matbench_discovery.structure import prototype
+
+NaCl = Structure(
+    lattice=[[2, 2, 0], [0, 2, 2], [2, 0, 2]],
+    species=["Na", "Cl"],
+    coords=[[0, 0, 0], [0.5, 0.5, 0.5]],
+)
+CsCl = Structure(
+    lattice=[[4, 0, 0], [0, 4, 0], [0, 0, 4]],
+    species=["Cs", "Cl"],
+    coords=[[0, 0, 0], [0.5, 0.5, 0.5]],
+)
+ZnO_zincblende = Structure(
+    lattice=[[2, 2, 0], [0, 2, 2], [2, 0, 2]],
+    species=["Zn", "O"],
+    coords=[[0, 0, 0], [0.25, 0.25, 0.25]],
+)
+ZnO_wurtzite = Structure(
+    lattice=Lattice.from_parameters(
+        a=3.8227, b=3.8227, c=6.2607, alpha=90, beta=90, gamma=120
+    ),
+    species=["Zn", "O", "Zn", "O"],
+    coords=[
+        [1 / 3, 2 / 3, 0],
+        [2 / 3, 1 / 3, 0.3748],
+        [2 / 3, 1 / 3, 1 / 2],
+        [1 / 3, 2 / 3, 1 / 2 + 0.3748],
+    ],
+)
+CaF2_fluorite = Structure(
```

Comment:
this looks off to me, this as this is a cubic lattice but then only given the primitive cell positions. I think you need to pass the primitive lattice here?

Reply:
thanks for giving it a look over. bit worried that there might be more issues, potentially here and/or in `moyopy`